agent/stats/container.go

```diff
@@ -59,13 +59,12 @@ func (container *CronContainer) StopStatsCron() {
 }
 
 // newCronContainer creates a CronContainer object.
-func newCronContainer(dockerID *string, name *string, dockerGraphPath string) *CronContainer {
+func newCronContainer(dockerID *string, dockerGraphPath string) *CronContainer {
```

Comment:
I'd feel a little safer if the first argument were a string, not a \*string unless there's a particular reason for making it a pointer.